Code/Conviva/Source/ConvivaConnector.swift

```diff
@@ -22,7 +22,7 @@ public struct ConvivaConnector {
     
     init(conviva: ConvivaEndpoints, player: THEOplayer, externalEventDispatcher: THEOplayerSDK.EventDispatcherProtocol? = nil) {
         self.endPoints = conviva
-        
+        self.convivaVPFDetector.player = player
         if let endPoints = self.endPoints {
```

Comment:
Why is this needed? `self.endPoints` cannot be nil here because it always equals to conviva, right?

Reply:
I updated the code